### PR TITLE
SDK-317: Extra value for context_dir setting after initializing new project

### PR DESCRIPTION
**Pull request by Kyrylo-Meretskyi**
[SDK-317](https://spryker.atlassian.net/browse/SDK-317)

### CODE REVIEW

File: src/Presentation/Console/Commands/InitProjectCommand.php [hunk 1]
```diff
@@ -130,10 +139,6 @@ protected function initializeSettingValues(array $settingEntities, bool $needsTo
                 default => (string)$values,
             };
 
-            if ($settingEntity->getStrategy() === SettingInterface::STRATEGY_MERGE) {
```
Comment (vol4onok) [inline src/Presentation/Console/Commands/InitProjectCommand.php:133]: why did you remove this strategy?

Comment (Kyrylo-Meretskyi) [reply src/Presentation/Console/Commands/InitProjectCommand.php:133]: Because we have the same code in `projectSettingManager::setSettings`

**Review by vol4onok — APPROVED** [review]
